client/my-sites/plan-features-2023-grid/hooks/npm-ready/data-store/use-plan-features-for-grid-plans.ts

```diff
@@ -33,12 +36,16 @@ const usePlanFeaturesForGridPlans: UsePlanFeaturesForGridPlans = ( {
 	intent,
 	selectedFeature,
 	showLegacyStorageFeature,
+	isInSignup,
 } ) => {
+	const highlightedFeatures = useHighlightedFeatures( intent ?? null, isInSignup );
+
 	return planSlugs.reduce( ( acc, planSlug ) => {
 		const planConstantObj = applyTestFiltersToPlansList( planSlug, undefined );
 		const isMonthlyPlan = isMonthly( planSlug );
+		const hasHightlightedFeatures = highlightedFeatures && highlightedFeatures.length > 0;
```

Comment:
We don't really need this. optional chaining below work equally right (`highlightedFeatures?.slice()` or `highlightedFeatures?.includes()` fort that two cases)? Although it does give a bit better context by having it 🤷‍♂️ 

No need to change anything.

Reply:
I removed this in any case as we updated the default return value to null, so just checking that.